content/en/docs/aboutjetstack/_index.md

```diff
@@ -1,22 +1,23 @@
 ---
 title: "About Jetstack"
 linkTitle: "About Jetstack"
-weight: 90
+weight: 500
 type: "docs"
 ---
 
-cert-manager was originally created by [Jetstack](https://jetstack.io), and
-a lot of the development is still sponsored by them.
+cert-manager was originally created by [Jetstack](https://jetstack.io), who
+still sponsor a lot of cert-manager development.
 
 Jetstack is a fast-growing Kubernetes professional services company that prides
 itself on helping startups and enterprises alike in their path to modern
 cloud-native infrastructure.
 
 They offer:
 
+- [Jetstack Secure](https://www.jetstack.io/jetstack-secure/), a machine identity management platform built on top of cert-manager.
```

Comment:
proposed:

`an enterprise machine identity management platform with support for cert-manager`.